lib/gollum/views/helpers.rb

```diff
@@ -2,6 +2,15 @@
 
 module Precious
   module Views
+    
+    module AppHelpers
+      def extract_page_dir(path)
+        return path if @page_dir.to_s.empty?
+        @extract_path ||= "#{Pathname.new(@page_dir).cleanpath.to_s}/"
+        path.start_with?(@extract_path) ? path[@extract_path.length..-1] : path
```

Comment:
I wonder if `@path_to_extract` would be a more expressive variable name than `@extract_path`.

Also, I think `path[@extract_path.length..-1]` could also be implemented using `slice` or `slice!`, which would look a little cleaner.